provisioning/Build.ps1

```diff
@@ -0,0 +1,87 @@
+<#
```

Comment:
How close is this to the scripts in the Window's repo? Should we come up with a strategy for reusing the bits that are the same?